capture.go

```diff
@@ -69,13 +71,14 @@ func (msg Capture) APIfy() APIMessage {
 	}
 
 	apified := CaptureInApi{
-		Type:           msg.Type,
-		Library:        library,
-		LibraryVersion: libraryVersion,
-		Timestamp:      msg.Timestamp,
-		DistinctId:     msg.DistinctId,
-		Event:          msg.Event,
-		Properties:     myProperties,
+		Type:             msg.Type,
+		Library:          library,
+		LibraryVersion:   libraryVersion,
+		Timestamp:        msg.Timestamp,
+		DistinctId:       msg.DistinctId,
+		Event:            msg.Event,
+		Properties:       myProperties,
+		SendFeatureFlags: msg.SendFeatureFlags,
```

Comment:
I might be getting this completely wrong, but sounds like this is sending a `SendFeatureFlags` property on the event, versus sending the feature properties?